docs/whats-new.md

```diff
@@ -11,6 +11,8 @@ of the [MetaMask developer page](https://metamask.io/developer/).
 
 ## April 2024
 
+- Documented [Snaps `onKeyringRequest` entry point](/snaps/reference/entry-points/#onkeyringrequest).
```

Comment:
I get a 404 when I click on the link at "Snaps `onKeyringRequest` entry point": https://github.com/MetaMask/metamask-docs/blob/dbfb78b25696b32de5ae7613d9f7d340431701b3/snaps/reference/entry-points/#onkeyringrequest

![Screenshot](https://github.com/MetaMask/metamask-docs/assets/32695229/34af8f28-c096-4121-bacb-c0f094e2e1c8)